module-2/backend/week_one.md

```diff
@@ -5,30 +5,69 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+<ul>
+  <li>get - Reads info at specified path</li>
+  <li>put - updates or replaces info </li>
+  <li> post - updates or replaces info</li>
+  <li> delete - deletes info</li>
+</ul>
+
 2. What is Sinatra?
+<p>Its a domain specific languang that allows for web application development</p>
+
 4. What is MVC?
+<p>MVC stands for Model View Controller</p>

```

Comment:
Yup! Recommend making sure that you're familiar with the responsibilities of each piece as well.